docs/spock_primer.adoc

```diff
@@ -81,7 +81,7 @@ the field at the very beginning of the `setupSpec()` method.)
 static final PI = 3.141592654
 ----
 
-Static fields should only be used for constants. Otherwise shared fields are preferable, because their semantics with
+Static fields should only be used for constants.Otherwise shared fields are preferable, because their semantics with
```

Comment:
An accidental change?

Reply:
Weird, must be some intellij autoformat going on. I only added L105+